hack/install-kcp-with-plugins.sh

```diff
@@ -19,11 +19,12 @@
 # This script installs KCP binaries and plugins to a folder of choice
 #
 # Arguments:
-# [--version release_version] set a specific KCP release version, default: latest
+# [--version release] set a specific KCP release version, default: latest
 # [--os linux|darwin] set a specific OS type, default: autodetect
 # [--arch amd64|arm64] set a specific architecture type, default: autodetect
-# [--folder installation_folder] sets the installation folder, default: $PWD/KCP
-# [--create-folder] create the instllation folder, if it does not exist
+# [--folder name] sets the installation folder, default: $PWD/KCP
+# [--create-folder] create the installation folder, if it does not exist
```

Comment:
(1) Are there use cases where users do NOT want the "create the installation folder, if it does not exist" behavior?  If not then I would remove this option as unnecessary complexity for users.

(2) If the answer to (1) is "yes", I would still argue that WANTING the "create the installation folder, if it does not exist" behavior is more common and thus should be the default.